City-of-Helsinki/drupal-helfi
# UHF-218: Admin theme adjustments

UHF-218 Hide site name from admin side, add breadcrumbs for admin theme, change site name to correct one

To test:

1.`make drush-cim`
2. Check that on admin side there is breadcrumb and that the site name is not visible anymore
3. Check that the site name is City of Helsinki
